modules/ibi-installer-configuration-config.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_mod-docs-content-type: REFERENCE
+[id="ibi-installer-configuration-config_{context}"]
+= ImageBasedConfig resource API specifications
```

Comment:
Same as [here](https://github.com/openshift/openshift-docs/pull/79610/files#r1754034359). Not sure about what we should replace the `resource API` with, but I think we should convey that this is the specification of the `image-based-config.yaml` configuration file.

Reply:
Updated to "Reference specifications for the image-based-config.yaml manifest"